src/Cache.ts

```diff
@@ -221,6 +219,28 @@ export class Cache {
             fileCache,
             file,
         );
+
+        // If there are no changes in any of the tasks, there's
+        // nothing to do, so just return.
+        if (Task.tasksListsIdentical(oldTasks, newTasks)) {
+            if (this.getState() == State.Warm) {
```

Comment:
`===` instead of `==` (should be a habit to always use that in JS *except for where you actually need `==`*)